go/gosec/bad_crypto/use_of_weak_crypto.yaml

```diff
@@ -0,0 +1,54 @@
+rules:
+  - id: use_of_md5
```

Comment:
The convention I see in `semgrep-rules` is to use `-` instead of `_` in rule IDs.

Reply:
ack